src/screens/permissions/index.js

```diff
@@ -55,10 +66,13 @@ const Permissions = (props) => {
         externalId: '',
         hidePngmeDialog: false, // if you want to hide the Pngme dialog, set this to true
       });
-      navigateToLoanScreen();
-    } else {
-      navigateToLoanScreen();
+    } else if (toggleCheckBox && user.pngmePermissionWasSelected) {
```

Comment:
Why not extracting the `toggleCheckBox` condition?  No strong opinion here, but I feel it would read better given how many similar booleans are around here.

Reply:
`toggleCheckBox` will return `true` as long as the `Use Pngme` checkbox is checked. This resulted in a weird behavior in the first permissions flow, if the user denies the SMS permission. As soon as they click `Deny` and navigate to the `Loan Screen`, the permission pops up again, if only `toggleCheckBox` is in the condition

The `user.pngmePermissionWasSelected` is set in the store, the first time the user checks the `Use Pngme` checkbox. This helps us in not showing the permissions popup on the `Loan Screen` in the first permission flow, if the user denied SMS permissions during the signup flow